commons/connector/upload/altoida/altoida_tapping_test_aggregate.avsc

```diff
@@ -0,0 +1,15 @@
+{
+  "namespace": "org.radarcns.connector.upload.altoida",
```

Comment:
I know that we skipped integrating the CSV_Export file. Are these part of already sent zip file?